packages/code-du-travail-api/src/server/routes/idcc/idcc.elastic.js

```diff
@@ -1,42 +1,49 @@
 function getIdccBody({ query }) {
+  const matchPhrasePrefix = {
+    match_phrase_prefix: {
```

Comment:
I would prefer leave that inside the query for readbility and I fond it more handy to  copy / paste in kibana too

Reply:
ok will do, was just trying to avoid duplication